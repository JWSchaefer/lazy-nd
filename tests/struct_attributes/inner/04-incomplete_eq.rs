use lazy_nd::lazy_nd;

#[lazy_nd(inner =)]
struct Test {}

fn main() {}
