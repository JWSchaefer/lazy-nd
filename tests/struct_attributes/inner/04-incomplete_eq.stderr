error: unexpected end of input, expected identifier or integer literal
 --> tests/struct_attributes/inner/04-incomplete_eq.rs:3:1
  |
3 | #[lazy_nd(inner =)]
  | ^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `lazy_nd` (in Nightly builds, run with -Z macro-backtrace for more info)
